README.md

```diff
@@ -7,44 +7,43 @@ panelGC effectively quantifies and monitors GC bias in hybridization capture seq
 amd64, arm64v8, ppc64le, s390x ([more info](https://github.com/docker-library/official-images#architectures-other-than-amd64))
 
 ## Dependencies
-- Nextflow (tested with 21.10.6)
 - Singularity (tested with 2.5.1)
 
-Containerized dependencies (no installation required):
-bedtools 2.30.0
-r-base 4.3.2 argparser 0.7.1 BiocManager 1.30.22 GenomicRanges 1.54.1 rtracklayer 1.62.0 tidyverse 2.0.0
-
```

Comment:
Info about dependencies was requested by reviewers and should be added back (remember to add Nextflow):

```
Containerized dependencies (no installation required):
Nextflow 23.04.0
bedtools 2.30.0
r-base 4.3.2 argparser 0.7.1 BiocManager 1.30.22 GenomicRanges 1.54.1 rtracklayer 1.62.0 tidyverse 2.0.0
```